pkg/btc-wallet/src/js/components/lib/balance.js

```diff
@@ -93,10 +95,25 @@ const Balance = () => {
             >
               {value}
             </Text>
-            <Text
-              fontSize={1}
-              color="orange"
-            >{`${sats}${unconfirmedString} sats`}</Text>
+            {scanning ? (
+              <Col alignItems="center">
+                <Row>
+                  <Text fontSize={1} color="orange">
+                    Balance will be updated shortly:
+                  </Text>
+                </Row>
+                <Row>
+                  <Text fontSize={1} color="orange">
+                    {scanProgress.main}/24 addresses scanned
```

Comment:
Why hardcoded 24?

Reply:
It seems to be the max number that comes back when the scan completes. Do you know if it will ever return higher than that number?

Reply:
@timlucmiptev may know better though